mappings/net/minecraft/entity/EntityType.mapping

```diff
@@ -12,6 +13,7 @@ CLASS aid net/minecraft/entity/EntityType
 		METHOD a create (Laik;)Laid$a;
 		METHOD a build (Ljava/lang/String;)Laid;
 		METHOD b disableSaving ()Laid$a;
+	CLASS aid$b EntityFactory
```

Comment:
Shouldn't it be `EntityTypeFactory` for consistency?

Reply:
No, it returns an Entity. Each EntityType will have an EntityFactory to instantiate its entities.

Reply:
Okay right 👍